### DISCUSSION TITLE
Lodestar Standup 2024-34 (September 3, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7054] [Standups]
### Meeting Info
[September 3, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/sep-03-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- v1.22.0 Planning
- Monthly PR Cleanup
- Open Discussion
- Team Updates
- Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by twoeths** [comment]
**v1.22.0**

there're so many code change due to electra, would be nice to cut a release for it soon. Need to discuss when and which PRs we want to include for v1.22.0

**Comment by philknows** [comment]
Looking to do a little monthly PR cleanup today, so if you have updates on your PRs, please leave notes on them async or during the meeting if you can make it!

**Comment by philknows** [comment]
Closing for #7068 . Notes can be found here: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#september-3-2024-planning-and-standup-meeting
